lib/local/platform/windows.js

```diff
@@ -1,6 +1,14 @@
 var os = require('os');
+var fs = require('fs');
 var path = require('path');
 var programFiles = os.arch() === "x64" ? process.env["ProgramFiles(x86)"] : process.env.ProgramFiles;
+var windowsDirectory = process.env.winDir;
+var systemApps = path.join(windowsDirectory, 'SystemApps');
+var microsoftEdgeDirectory = fs.readdirSync(systemApps).filter(function(folder) {
+    return folder.startsWith('Microsoft.MicrosoftEdge');
+  }).map(function(folder) {
+    return path.join(systemApps, folder);
+  })[0];
```

Comment:
I dynamically search for the path because the `MicrosoftEdge` folder has `8wekyb3d8bbwe` after it which has changed during the preview releases. Maybe we should just hard code the path:

`C:\Windows\SystemApps\Microsoft.MicrosoftEdge_8wekyb3d8bbwe\MicrosoftEdge.exe`

Then we'll get feedback quicker if anything changes. I don't know, for example, how the Edge technical preview gets installed.